charts/kyverno-policies/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 type: application
 name: kyverno-policies
-version: v2.4.0-rc3
-appVersion: v1.7.0-rc3
+version: v2.4.0
```

Comment:
@realshuting, why, when tagging a Kyverno release, are we also modifying the kyverno-policies Helm chart? The "kyverno" Helm chart I understand, but these are just the PSS policies which have not been changed in a while.